aspnetcore/release-notes/aspnetcore-6.0/samples/WebApp1/Program.cs

```diff
@@ -0,0 +1,249 @@
+app.Run();
+#endregion
+
+bool GetDesiredState()
+{
+    throw new NotImplementedException();
+}
+#elif JSONB
+#region snippet_jsonb
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorPages()
+            .AddNewtonsoftJson(options =>
+            { 
+                options.InputFormatterMemoryBufferThreshold = 48 * 1024;
+            });
+
+var app = builder.Build();
+#endregion
+
+if (!app.Environment.IsDevelopment())
+{
+    app.UseExceptionHandler("/Error");
+    app.UseHsts();
+}
+
+app.UseHttpsRedirection();
+app.UseStaticFiles();
+
+app.UseRouting();
+
+app.UseAuthorization();
+
+app.MapRazorPages();
+
+app.Run();
+#elif FDRS
+#region snippet_hdrs
+var builder = WebApplication.CreateBuilder(args);
+
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+app.Use(async (context, next) =>
+{
+    var hostHeader = context.Request.Headers.Host;
+    app.Logger.LogInformation("Host header: {host}", hostHeader);
+    context.Response.Headers.XPoweredBy = "ASP.NET Core 6.0";
+    await next.Invoke(context);
+    var dateHeader = context.Response.Headers.Date;
+    app.Logger.LogInformation("Response date: {date}", dateHeader);
+});
+
+app.Run();
+#endregion
+#elif IIS2
+#region snippet_iis2
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.Configure<IISServerOptions>(
+        options =>
+        {
+            options.MaxRequestBodySize = 64 * 1024;
+        }
+    );
+
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+app.Run();
+#endregion
+#elif HTTPLG
+#region snippet_httplg
+var builder = WebApplication.CreateBuilder(args);
+
+var app = builder.Build();
+app.UseHttpLogging();
+
+app.MapGet("/", () => "Hello World!");
+
+app.Run();
+#endregion
+#elif HTTPLG2
+#region snippet_httplg2
+using Microsoft.AspNetCore.HttpLogging;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddHttpLogging(logging =>
+{
+    // Customize HTTP logging.
+    logging.LoggingFields = HttpLoggingFields.All;
+    logging.RequestHeaders.Add("My-Request-Header");
+    logging.ResponseHeaders.Add("My-Response-Header");
+    logging.MediaTypeOptions.AddText("application/javascript");
+    logging.RequestBodyLogLimit = 4096;
+    logging.ResponseBodyLogLimit = 4096;
+});
+
+var app = builder.Build();
+app.UseHttpLogging();
+
+app.MapGet("/", () => "Hello World!");
+
+app.Run();
+#endregion
+#elif ICSF
+#region snippet_icsf
+var builder = WebApplication.CreateBuilder(args);
+builder.WebHost.ConfigureKestrel(serverOptions =>
+{
+    serverOptions.ConfigureEndpointDefaults(listenOptions => listenOptions.Use((connection, next) =>
+    {
+        var socketFeature = connection.Features.Get<IConnectionSocketFeature>();
+        socketFeature.Socket.LingerState = new LingerOption(true, seconds: 10);
+        return next();
+    }));
+});
+var app = builder.Build();
+app.MapGet("/", (Func<string>)(() => "Hello world"));
+await app.RunAsync();
```

Comment:
@pranavkm  this has multiple problems. Can you give me the correct code? This comes from [this blog entry](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-net-6-preview-4/#iconnectionsocketfeature)